testing/perl-net-mpd/APKBUILD

```diff
@@ -0,0 +1,44 @@
+# Automatically generated by apkbuild-cpan, template 2
+# Contributor: Leo <thinkabit.ukim@gmail.com>
+# Maintainer: Leo <thinkabit.ukim@gmail.com>
+pkgname=perl-net-mpd
+_pkgreal=Net-MPD
+pkgver=0.07
+pkgrel=0
+pkgdesc="Communicate with an MPD server"
+url="https://metacpan.org/release/Net-MPD/"
+arch="noarch"
+license="MIT"
+cpandepends=""
+cpanmakedepends="perl-module-build-tiny"
+cpancheckdepends=""
+depends="$cpandepends mpd"
+makedepends="perl-dev $cpanmakedepends"
+checkdepends="$cpancheckdepends"
```

Comment:
Because `cpancheckdepends` is empty, this can be removed as well.

Reply:
done